menelaos-front/src/app/footer/footer.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
As discussed earlier, all *spec.ts files can be safely deleted

Reply:
deleted all, added *.spec.ts to gitignore